SQL.script

```diff
@@ -1,4 +1,118 @@
-Create Database SocialMedia
```

Comment:
Right, I see. Too bad it won't do the same for the table.
There are times when I have some errors in the tables down below so I have to re-run the query. But the previous tables were already created so it did not allow. Then I have to DROP the Table to make the query work